docker-compose.yml

```diff
@@ -1,6 +1,27 @@
 version: '3.3'
 
 services:
+  lookout:
+    image: "srcd/lookout:latest"
+    network_mode: "host"
+    depends_on:
+      - postgres
+      - dummy
+    environment:
+      GITHUB_USER: ${GITHUB_USER}
+      GITHUB_TOKEN: ${GITHUB_TOKEN}
+    ports:
+      - "10301:10301"
+    entrypoint: ["/bin/sh"]
+    # sleep because container with db is up but the db itself doesn't accept connections yet
+    command: ["-c", "sleep 5 && lookout migrate && lookout serve ${REPO}"]
+    volumes:
+      - ./config.yml:/config.yml
+  dummy:
+    image: "smacker/dummy-analyzer:latest"
```

Comment:
Is this intended or did you forget to change this? If we want to include the dummy analyzer we should probably publish it as srcd. Or we could not include it as part of the compose, and write how setup an external analyzer. Which should be easy with the network mode host, I guess.

Reply:
I added it to have simple `docker-compose up` working. Without an analyzer the server wouldn't start.

We don't have any real analyzer for now and I don't see any value in publishing dummy as a project of src-d. I would just replace it by real analyzer from ML as soon as it's ready.

Reply:
It's not a big deal and I'm not going to block the PR, but I do see the value in doing all the required releases the same way, with the same credentials.

Reply:
I agree with Carlos and do not see a strong reason not to publish a dummy at the src-d account as part of the normal release process, esp if we are going to use it as part of the staging env.

Reply:
it requires changes to the release process :) we can do it in separate PR if we want.
Though I would suggest another solution to this problem. We can include dummy into lookout itself as a subcommand.

Reply:
> it requires changes to the release process
If that is anything more then adding ./docker/Dockerfile.dummy to `DOCKERFILES` - handling it in subsequent PR makes perfect sense. 

About the single image suggestion - at the same time I see value of going though the similar configuration as actual analyzers would go though on deployment.

Reply:
it is. Because our docker files aren't self-containing multistep. It requires the building of the binary on host machine. Which would lead to different release artifacts. We may want it or not. Let's track what we are doing and how it the separate issue.